metron-platform/metron-common/pom.xml

```diff
@@ -347,11 +347,6 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>commons-beanutils</groupId>
```

Comment:
Do you remember why we needed to get rid of this? There is still a relocation for `org.apache.commons.beanutils` in this pom.  Do we still need that relocation?

Reply:
Testing out if we can remove that altogether. I believe we simply didn't need it and/or it was added as an explicit dependency elsewhere closer to a leaf dep.

Reply:
The original reason was a conflict downstream, but I'm not 100% certain which module that was again.

Reply:
It looks like we actually still pull in beanutils 1.9.2 via commons-validator. I'm going to look into whether we can eliminate this entirely.

Reply:
@nickwallen I think if we reset this, we will probably need another full round of testing. This may cause random classpath issue downstream yet that we hadn't seen before. I think we serendipitously landed on the right beanutils version (1.9.3) via commons-validator. Given that this is a FB, I was thinking we could do another full round of validation on our cleanup PR with the final pom changes. What do you think?

Reply:
Yes, I think that is the right approach.  Let's get these bits in and see if we need to clean anything else up after.